enhancements/scheduling/scheduler-profiles.md

```diff
@@ -0,0 +1,208 @@
+...
+profiles:
+  - schedulerName: default-scheduler
+    plugins:
+      preScore:
+        disabled:
+        - name: "*"
+      score:
+        disabled:
+        - name: "*"
+``` 
+
+### Test Plan
+
+**Note:** *Section not required until targeted at a release.*
+
+Propagation of the expected profile settings can be verified through operator e2e's that check the right values are in 
+the config map that is currently mounted into the scheduler. Manual testing can also verify that these values are set as 
+expected through inspection of this config and logs from the scheduler.
+
+Testing that pods are scheduled to expected nodes is outside the scope of this enhancement, as that would be more focused 
+on testing of specific plugins which is difficult as the combination of plugins along with fluctuations in cluster states makes 
+not all scheduling decisions strictly deterministic.
+
+### Graduation Criteria
+
+**Note:** *Section not required until targeted at a release.*
+
+This field should technically be considered "beta" or "tech preview" as long as the underlying upstream API is in beta. 
+It should be marked as GA following the same promotion upstream (approximately upstream 1.23 -> OpenShift 4.10).
+
+The existing Policy field should immediately be marked as deprecated (effective in 4.7) and removed in the release that 
+Profiles is promoted to GA. This removal may come as a requirement in the rebase if it is also removed upstream (or require 
+a carry patch to continue support for this API).
+
+### Upgrade / Downgrade Strategy
+
+Upgrades may be an issue only for users of the old Policy API in the version that it is removed. We will provide both 
+options until the API is removed upstream (at least) for this reason. Otherwise, users will need to determine a reasonable 
+equivalent for their scheduler configuration out of the available options we provide (if applicable).
+
+Because this approach manages the raw profile values in our own operator, this should be more resilient to version skew 
+in the upstream scheduler plugin definitions and steps can be taken to react to any changes in the underlying config 
+before the release which would affect users.
+
+If more profile options are added, they may not be available in earlier versions causing downgrades to assume the default 
+scheduler configuration unless new profiles are backported.
+
+### Version Skew Strategy
+
+Many of the scheduler plugins are self-contained features, and the ones that do interact with or depend on matching features 
+in other components can be easily gated through this design. In general, minor version skew between the scheduler and other 
+components is not an issue.
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation
+History`.
+
+## Drawbacks
+
+* This is a more restrictive configuration option for the scheduler than what is currently supported. If users have a requirement 
```

Comment:
This might be an issue for developers who are adding new features (e.g. NUMA aware scheduling) where a custom configuration is a precondition.

Reply:
This is why I wasn't sold on the idea at first, because it seems counter to the work we are doing to enhance scheduling across teams and orgs. But talking to some of them, like the topology aware scheduling folks, I realized this isn't really a blocker. 

Like I mentioned in the other comment, anyone can still develop their own custom scheduler on OpenShift with a custom config, but this is meant to just restrict the core, default scheduler managed by our operator. It wouldn't be reasonable to imply our team will support any scheduler anyone develops by allowing them to run it through the default operator.

I will update the wording in this doc to loosen around the implication that "no" custom schedulers are allowed and make it clearer that this restriction only applys to the default scheduler.

Reply:
For devs it's not gonna be an issue :wink: